src/view/DialogInputBox.tsx

```diff
@@ -1,31 +1,57 @@
-import React, {useState} from "react";
+import React, {FormEvent} from "react";
 import {Textarea} from "nav-frontend-skjema";
 import {Hovedknapp} from "nav-frontend-knapper";
+import useFieldState from "../utils/useFieldState";
+import {DialogData} from "../utils/typer";
 
-function sendeNyMelding (props: { tekst: string }) {
-    console.log(props);
+function validerMelding(melding: string): string | null {
+    if (melding.length === 0) {
+        return "Melding må ha innhold.";
```

Comment:
Tror det er litt overlapp mellom PRer nå. Så se den andre PRen for kommentarer på dette.

Reply:
ok